src/components/10-atoms/input-file/index.scss

```diff
@@ -0,0 +1,173 @@
+.a-input-file {
+  @include typo(primary, x-small, medium);
+  position: relative;
+  display: inline-flex;
+  text-align: center;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 0 30px;
+  margin: 0;
+  min-height: 40px;
+  hyphens: auto;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+  text-transform: uppercase;
+  color: $color-prim-white;
+  background-color: $color-axa-blue;
+  border: solid 2px $color-axa-blue;
+  border-bottom-color: $color-prim-blue-deep-sapphire;
+
+  @include breakpoint($mediaquery-sm-up) {
+    @include _text(default)
+  }
+
+  &:hover,
+  &:active,
+  &:focus {
+    cursor: pointer;
+    text-decoration: none;
+    background-color: $color-prim-blue-deep-sapphire;
+    border-color: $color-prim-blue-deep-sapphire;
+  }
+
+  &__icon {
+    margin-right: 12px;
+    margin-left: 0px;
+  }
+
+  &__input {
+  	width: 0.001px;
+  	height: 0.001px;
+  	opacity: 0;
+  	overflow: hidden;
+  	position: absolute;
+  	z-index: -1;
+  }
+}
+
+.a-input-file--large {
+  min-height: 40px;
+  @include _text(default);
```

Comment:
It's very common to have @includes on the top of rules to be sure to override anything we imported. (preventing unusual side effects)